app/uk/gov/hmrc/gform/views/form/form.scala.html

```diff
@@ -96,20 +119,26 @@
 
     @formWithCSRF(action = page.formAction, args = 'class -> "js-form", 'autocomplete -> "on", 'novalidate -> "novalidate", 'id -> "gf-form") {
 
+    <div class="no-display">
+        <!-- this is here for the sole purpose of making it the default submit button, when fields of group exist -->
```

Comment:
Cannot this be twirl comment, so it is not send to the browser?

Reply:
Done